.idea/codeStyleSettings.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
What are these .idea files and do they have to be in remote?

Reply:
I wonder....